docs/reference-guide/server-stats.md

```diff
@@ -0,0 +1,36 @@
+---
+id: server-stats
+title: Server stats
+sidebar_label: Server stats
+---
+
+Memgraph supports multiple queries to get information about the instance that is being queried.
+
+## Instance version
+
+To get the version of the instance being queried, run the following query:
+
+```cypher
+SHOW VERSION
+```
+
+## Storage information
+
+Running the following query will return certain information about the storage of the current
+instance:
+
+```cypher
+SHOW STORAGE INFO
+```
+
+The result will contain following fields:
+
+| Field            | Description                                                |
+|------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| vertex_count     | Amount of vertices stored                                                                                                                    |
+| edge_count       | Amount of edges stored                                                                                                                       |
```

Comment:
Maybe `Number of xxx stored`? For me the amount is a general term that doesn't include the unit of measurement, but number suggest that the "unit" which is count of vertices/edges.